web/themes/custom/new_blog/src/scss/_global.scss

```diff
@@ -1,9 +1,14 @@
+html,
+body{
+  background-image: none;
+  background-color: #FFFFFF;
 
+}
 .container {
+  max-width: 1164px !important;
```

Comment:
it is a bad practice use !important in the short selectors. 
the best is to avoid using it at all